packages/icons/src/menu.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const MenuIcon = React.forwardRef<SVGSVGElement, IconProps>(
```

Comment:
isnt this the same as DotsHorizontalIcon?

Reply:
Not sure, will check.

Reply:
`DotsHorizontalIcon` is from `radix-ui` and it's slightly different.

Reply:
can we use one for both? Is design in figma using different icons here and in fonts manager? I think we only need one icon for this but I may be wrong, lets clarify it

Reply:
Yeah, I've replaced with `MenuIcon` in fonts